src/renderer/App.js

```diff
@@ -71,10 +71,18 @@ const App = (props) => {
   const [connected, setConnected] = globalContext.state.connected;
   const [focusDeviceDescriptor, setFocusDeviceDescriptor] =
     globalContext.state.focusDeviceDescriptor;
-  const [darkMode, setDarkMode] = globalContext.state.darkMode;
+  const [theme, setTheme] = globalContext.state.theme;
+  const darkMode = globalContext.state.darkMode;
   const [activeDevice, setActiveDevice] = globalContext.state.activeDevice;
 
-  setDarkMode(settings.get("ui.darkMode"));
+  // Migrate old `ui.darkMode` to new `ui.theme`
```

Comment:
Let’s move this to a function in a new file? App.js is already more crowded than it should be.